src/test/shell/bazel/BUILD

```diff
@@ -737,3 +737,18 @@ sh_test(
     ],
     tags = ["no_windows"],
 )
+
+sh_test(
+    name = "jdeps_test",
+    size = "medium",
+    srcs = ["jdeps_test.sh"],
+    data = [
+        ":jdeps_modules.golden",
+        ":jdeps_class_blacklist.txt",
+        ":test-deps",
+        "//src:embedded_jdk",
```

Comment:
Once we minimize the jdk it will not contain a `jdeps` tool?

Reply:
The dependency here will change (probably just the name). So this test will depend on the allmodules JDK.